npm/config/config.go

```diff
@@ -3,16 +3,23 @@ package npmconfig
 const (
 	defaultResyncPeriod  = 15
 	defaultListeningPort = 10091
-
+	defaultGrpcPort      = 10092
```

Comment:
This makes me think, we should add a default network policy within kube-system NS to always allow these two ports to NPM pods.